windows-driver-docs-pr/wdk22h2-SupportMatrix.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Windows 11, version 22H2 Support Matrix
+description: Page articulates the support matrix for the new Windows Driver Kit (WDK)
+keywords:
+- Windows Driver Kit
+- WDK
+- drivers
+- Support Matrix
+ms.date: 05/23/2022
+---
+
+# Windows 11, Version 22H2 Support Matrix
+
+The WDK/EWDK for Windows 11, version 22H2 will align its support matrix to match [Windows 11 new hardware requirements.](https://docs.microsoft.com/en-us/windows/whats-new/windows-11-requirements) 
+
+Windows 11, version 22H2 WDK/EWDK <b>will</b> support: 
+
+•	Building and testing Kernel Mode drivers for X64 and ARM64 </br>

```

Comment:
You should probably use the Markdown format for bulleted lists on all bulleted items.

```suggestion
* Building and testing Kernel Mode drivers for X64 and ARM64
```